test/api/test_default_schema.cpp

```diff
@@ -0,0 +1,22 @@
+#include "catch.hpp"
+#include "test_helpers.hpp"
+
+using namespace duckdb;
+using namespace std;
+
+TEST_CASE("Test Default Schema", "[api]") {
+	unique_ptr<QueryResult> result;
+	DuckDB db(nullptr);
+	Connection con(db);
+
+	REQUIRE_NO_FAIL(con.Query("CREATE SCHEMA s1;"));
```

Comment:
Rather than creating a C++ test, please create a sqllogic test for SQL-only tests. We prefer to avoid C++ tests unless absolutely required (e.g. directly testing C++ API's or testing more complex behaviors involving multi-threading). See e.g. the [test_schema.test file here](https://github.com/cwida/duckdb/blob/master/test/sql/catalog/test_schema.test)

Reply:
Thanks, didn't know that! Moved to SQL-only test and added a bunch of more tests.